pkg/driver/clients/grpc_client_test.go

```diff
@@ -67,11 +78,24 @@ func TestGRPCClientWithServerUnreachable(t *testing.T) {
 		record.RequestedDuration != uint32(testRuntimeSpecs.Runtime*1000) ||
 		record.StartTime == 0 ||
 		record.ResponseTime == 0 ||
-		success != false ||
+		success == false ||
```

Comment:
From the test name, it is testing the unreachable server. So, we expect it to fail. Hence, the condition for failing the test should be `success == true`. Are you sure that we need to change the condition here?